src/omnipkg/apis/local_bridge.py

```
@@ -2,271 +2,412 @@
+            port = self._get_port()
+            print(f"✅ Web bridge already running on port {port}")
+            print(f"🌍 Dashboard: {PRIMARY_DASHBOARD}#{port}")
+            return 0
+        
+        print("🚀 Starting web bridge...")
+        
+        # Start this very file as a module in a new process
+        cmd = [sys.executable, "-m", "omnipkg.apis.local_bridge"]
+        
+        with open(self.log_file, 'w') as log:
+            process = subprocess.Popen(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)